src/test/java/ua/softserveinc/tc/util/RateUtils.java

```diff
@@ -0,0 +1,18 @@
+package ua.softserveinc.tc.util;
+
+import ua.softserveinc.tc.entity.Rate;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class RateUtils {
+
+    public static List<Rate> getListOfRates() {
+        Rate rate1 = new Rate();
+        Rate rate2 = new Rate();
+        Rate rate3 = new Rate();
```

Comment:
You're right, I shouldn't create this variables instead I should mock this, but I try to use JUnitParams  and miss this two classes (RateDaoTest.java, EventDaoTest.java) when refactoring. Test method uses this variables just to pass to the method. 
I will rewrite it.